client/my-sites/site-settings/jetpack-module-toggle.jsx

```diff
@@ -31,7 +31,7 @@ class JetpackModuleToggle extends Component {
 	};
 
 	static propTypes = {
-		siteId: PropTypes.number.isRequired,
```

Comment:
Another alternative that came to my mind was making the JetpackModuleToggle don't depend on a passed down prop, and have it as a connected prop instead based on `getSelectedSiteId`. But this would also mean that for a render without warnings, the prop should be declared as non required.